src/bin/faf

```diff
@@ -74,8 +75,15 @@ def main():
     cmdline_parser = CmdlineParser(toplevel=True,
                                    desc="Perform a FAF action",
                                    prog=sys.argv[0])
+    cmdline_parser.add_argument("-V", "--version", action="store_true",
+                                help="show version information and exit")
     cmdline = cmdline_parser.parse_args()
 
+    # print version and quit
+    if cmdline.version:
+        print("ABRT Analytics v{}".format(pyfaf.__version__))
+        exit(0)
```

Comment:
https://docs.python.org/3.7/library/constants.html#exit

There’s a comment above that, saying that `exit()` is more for the interactive interpreter. https://github.com/abrt/abrt/blob/5cc79410beef777509a50ccf43c108d102a5d075/src/plugins/abrt-action-install-debuginfo.in#L54 even suggests that it can cause trouble sometimes.